developers_chamber/ecs_utils.py

```diff
@@ -463,39 +464,48 @@ def run_task_and_wait_for_success(cluster, task_definition, command, name, succe
     task_id = task.split('/')[-1]
 
     LOGGER.info("Running task: '{}'".format(task))
-
     wait_for_task_to_stop(cluster=cluster, task=task, timeout=timeout, region=region, ecs_client=ecs_client)
-
     response = ecs_client.describe_tasks(cluster=cluster, tasks=[task])
 
     UNDEFINED = 'UNDEFINED'
 
     task_stop_code = response['tasks'][0].get('stopCode', UNDEFINED)
     LOGGER.info("Task stop code: '{}'".format(task_stop_code))
 
+    container_response = {}
```

Comment:
problém byl v tom, že se pouštěly dva tasky nginx a admin a vybíralo to náhodně result nginx a ne admina (podle toho jaké pořadí výsledku kontejnerů se vrátilo)